src/main/java/net/starlark/java/eval/ParamDescriptor.java

```diff
@@ -33,6 +33,7 @@
   private final boolean named;
   private final boolean positional;
   private final List<Class<?>> allowedClasses; // non-empty
+  private final boolean allowedClassesContainObject;
```

Comment:
This is an awkward (and awkwardly-named) workaround for the artificial constraint that `ParamDescriptor.allowedClasses` must be a non-empty list.

Instead, let's remove the artificial constraint! Add a comment that an empty `allowedClasses` list means that any type is allowed. And in the `ParamDescriptor` constructor, transform an `allowedClasses` containing `Object.class` into an empty list.

An empty `allowedClasses` list will automatically avoid the `isInstance` call in `checkParamValue()`.

Reply:
Done.